app/helpers/hosts_helper.rb

```diff
@@ -183,7 +183,7 @@ def multiple_actions_select
   end
 
   def date(ts = nil)
-    return _("%s ago") % (time_ago_in_words ts) if ts
+    return date_time_relative_value(ts) if ts
     _("N/A")
```

Comment:
You can also consider
```suggestion
    ts ? date_time_relative_value(ts) : _("N/A")
```

Reply:
Thank you, I will.